internal/databases/postgres/paged_file_verifier.go

```diff
@@ -201,21 +203,26 @@ func VerifyPagedFileBase(path string, fileInfo os.FileInfo, pagedFile io.Reader)
 // verifyPageBlocks verifies provided page blocks from the pagedBlocks reader
 func verifyPageBlocks(path string, fileInfo os.FileInfo, pageBlocks io.Reader,
 	blockNumbers []uint32) (corruptBlockNumbers []uint32, err error) {
-	if _, ignored := ignoredFileNames[fileInfo.Name()]; ignored || !isPagedFile(fileInfo, path) {
+	if _, ignored := ignoredFileNames[fileInfo.Name()]; ignored || !isChecksumValidatableFile(fileInfo, path) {
 		_, err = io.Copy(io.Discard, pageBlocks)
 		return nil, err
 	}
 	for _, blockNo := range blockNumbers {
 		corrupted, err := verifySinglePage(path, blockNo, pageBlocks)
+		if corrupted {
+			corruptBlockNumbers = append(corruptBlockNumbers, blockNo)
+		}
 		if err == io.EOF {
 			break
 		}
+		if err == io.ErrUnexpectedEOF {
+			corruptBlockNumbers = append(corruptBlockNumbers, blockNo)
+			tracelog.WarningLogger.Printf("verifyPageBlocks: %s invalid file size %d\n", path, fileInfo.Size())
```

Comment:
why do we log anything here but do not log above?

Reply:
> why do we log anything here but do not log above?

I didn’t quite understand what you mean?
I only see the block with io.EOF without logging (but it triggers after processing each file and we just skip it).